doc/htmldoc/neurons/model_details/IAF_Integration_Singularity.ipynb

```diff
@@ -0,0 +1,945 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Handling time-constant singularities in integrate-and-fire neurons\n",
+    "\n",
+    "This notebook details how NEST handles the numerical instability of the exact integration propagator matrix $P = e^{A h}$ which arise if $\\tau_m\\approx \\tau_{\\text{syn}}$. For an overview over exact integration of integrate-and-fire neuron subthreshold dynamics, please see `exact-integration.rst`.\n",
```

Comment:
```suggestion
    "This notebook details how NEST handles the numerical instability of the exact integration propagator matrix $P = e^{A h}$ which arise if $\\tau_m\\approx \\tau_{\\text{syn}}$. For an overview over exact integration of integrate-and-fire neuron subthreshold dynamics, please see [exact-integration](model_details/exact-integration.rst).\n",
```

Reply:
It's possible to link to rst files from notebooks  using the markdown syntax

Reply:
Thanks for the hint, I have accepted it! Will this link to the rst file be re-written into a link to the corresponding HTML when documentation is rendered?